libs/auth-api-lib/seeders/20201128002958-add-scope-for-dev.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
```

Comment:
Why have twice the same migration?

Reply:
Two different files

Reply:
It is better to be in one migration, since I think that is the atomicity/transaction boundary.

Reply:
Merged into one